common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -2,42 +2,69 @@
 import unittest
 
 from xmodule import graders
-from xmodule.graders import Score, aggregate_scores
+from xmodule.graders import ProblemScore, AggregatedScore, aggregate_scores
```

Comment:
*Note to reviewers:* The changes in this test file are mostly just formatting changes, per new conventions.  The only substantive change is going from `Score` to `ProblemScore` or `AggregatedScore`, as appropriate.